### DISCUSSION TITLE
关于预览图管理功能及其实现方法的征集

### DISCUSSION THREAD

**Discussion by 5Jbgr** [5Jbgr/carolineModCtrl#23] [General]
1. 预览图管理希望实现的功能
2. 管理模块的程序流程（思路、界面，建议，方向 。。。）

**Comment by SinDou1024** [comment]
1.模组信息编辑的时候，可以直接拖入图片当做预览图，图片的存放和绑定还是照旧。
2.输入网址如果是n网地址，也许能获取n网里的预览图？（只是个想法）

**Reply by 5Jbgr** [reply]
感谢你的建议，
目前在启用模组信息编辑的状态下，是可以通过将图片拖拽到显示图片所在的位置进行添加图片的（记得点保存）
n网地址目前会考虑尝试，但是n网多数需要登录后才能访问，获取可能有些困难
